Automattic/jetpack
# Update VideoPress module to use the standard media endpoint

There has been a rewrite of the /media/new endpoint for the wpcom api to account for Jetpack-based VideoPress uploads. This approach is a bit more asynchronous regarding the uploading of new video and should have fewer race conditions associated to it.

## Testing

In order to test this, do the following:

1) Enable Jetpack
2) Signup for a Jetpack pro or business plan
3) Go to Media -> New
4) Upload a video
  - Processing can take a minute or two and is not reflected until the page is refreshed
5) After a minute or 2 refresh the page and verify that the video is present and has a proper thumbnail.
